src/main/java/Ui.java

```diff
@@ -0,0 +1,95 @@
+package duke;
+
+/**
+ * Deals with interactions with the user
+ */
+public class Ui {
+    Ui() {
+    }
+
+    /**
+     * Prints the welcome statement.
+     */
+    public void start() {
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    /**
+     * Prints the goodbye statement.
+     */
+    public void finalBye() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    /**
+     * Prints the error message for loading error.
+     */
+    public void showLoadingError() {
+        System.out.println("System data could not be loaded!");
+    }
+
+    /**
+     * Prints the error message for empty description.
+     */
+    public void emptyInput() {
+        System.out.println(" ☹ OOPS!!! The description of a todo cannot be empty.");
+    }
+
+    /**
+     * Prints the error message for invalid commands.
+     */
+    public void doNotUnderstand() {
+        System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+    }
+
+    /**
+     * Prints the message for marking a task as done.
+     * @param task is the task which is marked as done.
+     */
+    public void markDone(Task task) {
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println(task);
+    }
+
+    /**
+     * Prints the message for marking a task as done.
+     *  @param task is the task which is marked as undone.
+     */
```

Comment:
```suggestion
    /**
     * Prints the message for unmarking a task.
     *  @param task is the task which is unmarked.
     */
```

I think you might have a careless mistake here as the header does not describe correctly what the method does. In this case it is unmarking the task instead of marking.